dashboards/use.libsonnet

```diff
@@ -74,9 +74,21 @@ local g = import 'grafana-builder/grafana.libsonnet';
         g.row('Storage')
         .addPanel(
           g.panel('Disk Capacity') +
-          g.queryPanel(|||
-            sum(max(node_filesystem_size{fstype=~"ext[24]"} - node_filesystem_avail{fstype=~"ext[24]"}) by (device,%(podLabel)s,namespace)) by (%(podLabel)s,namespace) / scalar(sum(max(node_filesystem_size{fstype=~"ext[24]"}) by (device,%(podLabel)s,namespace))) * on (namespace, %(podLabel)s) group_left(node) node_namespace_pod:kube_pod_info:
-          ||| % $._config, '{{node}}', legendLink) +
+          g.queryPanel(
+            |||
```

Comment:
This multi line string seems a bit too spacey to me. Mind if we could put some more lines in one? Maybe
```
node_filesystem_size{%(fstypeSelector)s} - node_filesystem_avail{%(fstypeSelector)s}
```

Reply:
sounds good, yes, this was me in "dissection" mode ;-) I will re-join these expressions.